package.json

```diff
@@ -73,12 +73,13 @@
     "coveralls": "cat ./test/coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
     "dev": "rimraf ./lib && babel -w src --out-dir lib --copy-files",
     "lint": "eslint src/**",
-    "plaintest": "mocha --check-leaks -b -R spec test/index.js && npm run tape",
+    "plaintest": "mocha -t 10000 -b -R spec test/index.js && npm run tape",
     "prepublish": "npm run babel",
     "pre_test": "npm run lint",
     "tape": "node test/tape/index.js | tap-spec",
     "debug_tape": "node-debug test/tape/index.js",
     "test": "npm run pre_test && istanbul --config=test/.istanbul.yml cover node_modules/mocha/bin/_mocha -- --check-leaks -t 10000 -b -R spec test/index.js && npm run tape",
+    "test:oracledb": "docker rmi -f --no-prune knex-test-oracledb && docker build -f scripts/oracle-tests-Dockerfile --tag knex-test-oracledb . && docker run -i -t knex-test-oracledb",
```

Comment:
I think I will change this to be named:  `oracledb:test` I'm making PR tomorrow for running mssql tests too (they seem to be failing currently though)

Reply:
This is awesome. 👍

Reply:
I'll update task name in mssql PR